drivers/display/Kconfig.sdl

```diff
@@ -88,6 +88,12 @@ config SDL_DISPLAY_TRANSPARENCY_GRID_CELL_COLOR_2
 	help
 	  The color of the even cells in the transparency grid.
 
+config SDL_DISPLAY_COLOR_TINT
```

Comment:
new Kconfig needs to be mentioned in release notes

Reply:
Like so?